reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -56,6 +57,17 @@
     openshift_hosted_registry_pullthrough: true
     openshift_hosted_registry_acceptschema2: true
     openshift_hosted_registry_enforcequota: true
+    # metrics
     openshift_hosted_metrics_storage_kind: dynamic
     openshift_metrics_hawkular_hostname: "hawkular-metrics.apps.{{ public_hosted_zone }}"
```

Comment:
dynamic is the default value or which var? somehow the review didn't post clean but i think maybe you mean the hawkular_hostname